strawberry/experimental/pydantic/fields.py

```diff
@@ -26,13 +26,10 @@
 except ImportError:
     import sys
 
-    # python < 3.9 does not have GenericAlias (list[int], tuple[str, ...] and so on)
+    # python < 3.10 does not have GenericAlias (list[int], tuple[str, ...] and so on)
     # we do this under a conditional to avoid a mypy :)
-    if sys.version_info < (3, 9):
-        TypingGenericAlias = ()
-    else:
-        raise
     if sys.version_info < (3, 10):
+        TypingGenericAlias = ()
```

Comment:
<img width="518" alt="CleanShot 2023-08-09 at 10 50 22@2x" src="https://github.com/strawberry-graphql/strawberry/assets/667029/a2006c3f-fb61-4c4a-9df5-35409048db3c">

this seems to be available for me

Reply:
yea, separated the `TypeUnionAlias` and `TypeGenericAlias` version checks in their own try-catch